pkg/morph/subscriber/subscriber.go

```diff
@@ -68,6 +68,7 @@ func (s *subscriber) NotificationChannels() NotificationChannels {
 }
 
 var (
+	ErrStaleNode = errors.New("RPC node is not yet up to date")
```

Comment:
public err with no comments. not critical but usually we do not do that

Reply:
Fixed.